src/components/molecules/EditRoomForm/EditRoomForm.tsx

```diff
@@ -37,6 +39,7 @@ interface EditRoomFormProps extends AdminSidebarFooterProps {
   venueVisibility?: RoomVisibility;
 }
 
+// @debt Is this form used anywhere anymore? Remove if not so
```

Comment:
doesn't seem to be used. Delete it here or i'll do it in my PR

Reply:
You can do it. After all, you might be more familiar with all those forms  than I am.